### DISCUSSION TITLE
Any plans to integrate with graphql?

### DISCUSSION THREAD

**Discussion by tenzan** [nuxt/nuxt#9646] [Ideas]
Hi!

Any plans to support GraphQL out of the box like gatsby or gridsome?

**Comment by kissu** [comment]
This is not planned out of the box, as far as I know.
[@nuxtjs/apollo](https://github.com/nuxt-community/apollo-module) is pretty much the go-to starter to have a quick working GraphQL configuration.

If enough people ask for it, we maybe consider adding it to the CLI as an optional checkbox. 🤔
